components.json

```diff
@@ -79,7 +79,7 @@
 			"owner": "dellagustin"
 		},
 		"abnf": {
-			"title": "Augmented Backus–Naur form",
+			"title": "ABNF",
```

Comment:
Maybe this is better as an alias?

Reply:
Agreed.

Reply:
Ahh, no. Looking at it again, I remembered the reason why I didn't add this as an alias in the first place: I can't. An alias title requires at least one alias (of the language id) or else the alias test will fail.

Reply:
Ok, so that makes sense but then why prefer `ABNF` over the long-form?

Reply:
lol I need to read everything before commenting. I don't know anything about these languages so I'm not sure whether the abbreviations are more common. I'm ok with this if you are

Reply:
I was thinking that if I searched for this language, I'd put "ABNF" into my search box. But for example, HighlightJS offers it under the long name. 

We don't really have rules for language names and the use of alias titles.

Reply:
Alright, your last comment just reached me... 
I'll just go with the abbreviations for now.